### ISSUE TITLE
conda fails with FileNotFoundError after upgrading to requests-2.11.1-py35_0

### ISSUE THREAD

**Issue by ciupicri** [conda/conda#3947]
Current conda install:

               platform : win-64
          conda version : 4.2.13
       conda is private : False
      conda-env version : 4.2.13
    conda-build version : 2.0.10
         python version : 3.5.2.final.0
       requests version : 2.12.1
       root environment : E:\Program Files\Anaconda3  (writable)
    default environment : E:\Program Files\Anaconda3
       envs directories : E:\Program Files\Anaconda3\envs
          package cache : E:\Program Files\Anaconda3\pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : None
           offline mode : False

`$ E:\Program Files\Anaconda3\Scripts\conda-script.py install --override-channels --channel file:///A:/pkgs/free --no-channel-priority pyodbc`

    Traceback (most recent call last):
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\exceptions.py", line 479, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\cli\main.py", line 145, in _main
        exit_code = args.func(args, p)
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\cli\main_install.py", line 80, in execute
        install(args, parser, 'install')
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\cli\install.py", line 238, in install
        prefix=prefix)
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\api.py", line 24, in get_index
        index = fetch_index(channel_urls, use_cache=use_cache, unknown=unknown)
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\fetch.py", line 300, in fetch_index
        repodatas = [(u, f.result()) for u, f in zip(urls, futures)]
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\fetch.py", line 300, in <listcomp>
        repodatas = [(u, f.result()) for u, f in zip(urls, futures)]
      File "E:\Program Files\Anaconda3\lib\concurrent\futures\_base.py", line 398, in result
        return self.__get_result()
      File "E:\Program Files\Anaconda3\lib\concurrent\futures\_base.py", line 357, in __get_result
        raise self._exception
      File "E:\Program Files\Anaconda3\lib\concurrent\futures\thread.py", line 55, in run
        result = self.fn(*self.args, **self.kwargs)
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\fetch.py", line 75, in func
        res = f(*args, **kwargs)
      File "E:\Program Files\Anaconda3\lib\site-packages\conda\fetch.py", line 117, in fetch_repodata
        timeout=(6.1, 60))
      File "E:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 501, in get
        return self.request('GET', url, **kwargs)
      File "E:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 488, in request
        resp = self.send(prep, **send_kwargs)
      File "E:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 641, in send
        r.content
      File "E:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 772, in content
        self._content = bytes().join(self.iter_content(CONTENT_CHUNK_SIZE)) or bytes()
      File "E:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 705, in generate
        chunk = self.raw.read(chunk_size)
    AttributeError: 'FileNotFoundError' object has no attribute 'read'

**Comment by wikiped** [comment]
Same issue with `requests version : 2.12.4`:

    Current conda install:

                   platform : win-64
              conda version : 4.2.13
           conda is private : False
          conda-env version : 4.2.13
        conda-build version : 2.0.12
             python version : 3.5.2.final.0
           requests version : 2.12.4

**Comment by ostrokach** [comment]
I get the same error on Linux (`condaforge/linux-anvil:latest` docker container):

```
(root) [root@af8f8e0450ca recipe_root]# conda install --use-local pdb2pka
Fetching package metadata ..........An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



Current conda install:

               platform : linux-64
          conda version : 4.2.13
       conda is private : False
      conda-env version : 4.2.13
    conda-build version : 2.1.2
         python version : 3.5.3.final.0
       requests version : 2.12.5
       root environment : /opt/conda  (writable)
    default environment : /opt/conda
       envs directories : /opt/conda/envs
          package cache : /opt/conda/pkgs
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : /root/.condarc
           offline mode : False

`$ /opt/conda/bin/conda install --use-local pdb2pka`
```

The Traceback is:

    Traceback (most recent call last):
      File "/opt/conda/lib/python3.5/site-packages/conda/exceptions.py", line 479, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "/opt/conda/lib/python3.5/site-packages/conda/cli/main.py", line 145, in _main
        exit_code = args.func(args, p)
      File "/opt/conda/lib/python3.5/site-packages/conda/cli/main_install.py", line 80, in execute
        install(args, parser, 'install')
      File "/opt/conda/lib/python3.5/site-packages/conda/cli/install.py", line 238, in install
        prefix=prefix)
      File "/opt/conda/lib/python3.5/site-packages/conda/api.py", line 24, in get_index
        index = fetch_index(channel_urls, use_cache=use_cache, unknown=unknown)
      File "/opt/conda/lib/python3.5/site-packages/conda/fetch.py", line 300, in fetch_index
        repodatas = [(u, f.result()) for u, f in zip(urls, futures)]
      File "/opt/conda/lib/python3.5/site-packages/conda/fetch.py", line 300, in <listcomp>
        repodatas = [(u, f.result()) for u, f in zip(urls, futures)]
      File "/opt/conda/lib/python3.5/concurrent/futures/_base.py", line 398, in result
        return self.__get_result()
      File "/opt/conda/lib/python3.5/concurrent/futures/_base.py", line 357, in __get_result
        raise self._exception
      File "/opt/conda/lib/python3.5/concurrent/futures/thread.py", line 55, in run
        result = self.fn(*self.args, **self.kwargs)
      File "/opt/conda/lib/python3.5/site-packages/conda/fetch.py", line 75, in func
        res = f(*args, **kwargs)
      File "/opt/conda/lib/python3.5/site-packages/conda/fetch.py", line 117, in fetch_repodata
        timeout=(6.1, 60))
      File "/opt/conda/lib/python3.5/site-packages/requests/sessions.py", line 501, in get
        return self.request('GET', url, **kwargs)
      File "/opt/conda/lib/python3.5/site-packages/requests/sessions.py", line 488, in request
        resp = self.send(prep, **send_kwargs)
      File "/opt/conda/lib/python3.5/site-packages/requests/sessions.py", line 641, in send
        r.content
      File "/opt/conda/lib/python3.5/site-packages/requests/models.py", line 781, in content
        self._content = bytes().join(self.iter_content(CONTENT_CHUNK_SIZE)) or bytes()
      File "/opt/conda/lib/python3.5/site-packages/requests/models.py", line 714, in generate
        chunk = self.raw.read(chunk_size)
    AttributeError: 'FileNotFoundError' object has no attribute 'read'

If I set the `--use-index-cache` flag, I can install the package without error.

**Comment by ostrokach** [comment]
I only get this error when using `requests>=2.12` (i.e. 2.11 and 2.12). 

Using `requests==2.11` everything is fine.

**Comment by kalefranz** [comment]
Fixed in later versions of 4.2.

**Comment by ostrokach** [comment]
This error is back in `conda 4.2.13`...
